main.py

```diff
@@ -1,3 +1,4 @@
+# [Q]: this MenuItem should not be imported AFAIK

```

Comment:
MenuItem here calculates the resources based on the user's choice. That's why it has been imported.